ui/ducks/platform-notifications/platform-notifications.js

```diff
@@ -0,0 +1,7 @@
+// Selectors
+export const getPlatformNotificationsList = (state) =>
+  state.platformNotifications.notificationsList;
```

Comment:
Hmm, I guess this is still a work in progress? I'm not seeing this state anywhere

Reply:
yes, you are right about that